### PR TITLE
Upgrade to 1.1.7

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Not up to date*
- *Implement LDAP*
- *Fix https://github.com/YunoHost-Apps/pleroma_ynh/issues/81 using OTP version*
- *Fix https://github.com/YunoHost-Apps/pleroma_ynh/issues/28 with ldap implementation*
- *Fix https://github.com/YunoHost-Apps/pleroma_ynh/issues/92*

## Solution
- [x] *Upgrade to 1.1.7*
- [x] *Error during installation: https://git.pleroma.social/pleroma/pleroma/issues/1099*
- [ ] *Error during installation: https://git.pleroma.social/pleroma/pleroma/issues/1289*
- [x] *Error during installation: https://git.pleroma.social/pleroma/pleroma/issues/1343*
- [x] *2nd Error during https://git.pleroma.social/pleroma/pleroma/issues/1327 since recent version*
- [ ] *Implement LDAP. https://git.pleroma.social/pleroma/pleroma/issues/1195*
- [ ] *Include elixir eldap module in the OTP release https://git.pleroma.social/pleroma/pleroma/issues/1313*
- [x] *Use OTP version.*
- [ ] *Manage upgrade from previous 0.9.99999 using https://docs-develop.pleroma.social/backend/installation/migrating_from_source_otp_en/*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pleroma_ynh%20PR87%20(yalh76)/)

### CODE REVIEW

**Comment by Josue-T** [comment]
Hello,

What is the state of this PR ? Is it ready to merge ?

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For user friendly details about Pleroma: [see here](https://blog.soykaf.com/post
 - Tootdon (Android + iOS)
 - Tootle (iOS)
 
-**Shipped version:** 0.9.99999
+**Shipped version:** 1.0.1
```
Comment (Josue-T) [inline README.md]: Maybe you can also update the README to say that LDAP is now supported. :smile:

**Comment by yalh76** [comment]
> 
> 
> Hello,
> 
> What is the state of this PR ? Is it ready to merge ?

No, the PR is not ready. LDAP authentication not implemented and still have to manage the upgrade to the OTP version

**Comment by Thatoo** [comment]
I'm so excited about it.
Congratulation and thank you again @yalh76 .

**Comment by Thatoo** [comment]
As for https://git.pleroma.social/pleroma/pleroma/issues/1099, you could add a check mark to
- [x] Use OTP version

File: README.md (hunk 1 above)
Comment (yalh76) [reply README.md]: It's planned to update the readme, just waiting to have ldap implemented in the package ^^

**Comment by yalh76** [comment]
> 
> 
> As for https://git.pleroma.social/pleroma/pleroma/issues/1099, you could add a check mark to
> 
>     * [x]  Use OTP version
>       couldn't you?

Done ;) 
* But still an issue during install...
* Not tested ldap authentication
* Big work to manage the upgrade fro previous verson to OTP version: https://docs-develop.pleroma.social/migrating_from_source_otp_en.html#moving-content-out-of-the-application-directory

**Comment by yalh76** [comment]
But moving forward. now stucked by https://git.pleroma.social/pleroma/pleroma/issues/1313 ^^

**Comment by yalh76** [comment]
Switched to a new PR: https://github.com/YunoHost-Apps/pleroma_ynh/pull/93

**Pull request closed without merging**